airflow/providers/amazon/aws/operators/sns.py

```diff
@@ -33,8 +33,13 @@ class SnsPublishOperator(BaseOperator):
     :type target_arn: str
     :param message: the default message you want to send (templated)
     :type message: str
+    :param subject: the message subject you want to send (templated)
+    :type subject: str
+    :param message_attributes: the message attributes you want to send as a flat dict (data type will be
+        determined automatically)
+    :type message_attributes: mapping
```

Comment:
```suggestion
    :type message_attributes: dict
```

Reply:
`mapping` is not a valid type. You can explain the mapping in detail in the description. The type should be valid.